package.json

```diff
@@ -35,8 +36,7 @@
       "git add README.md"
     ],
     "*.{ts,tsx}": [
-      "prettier-standard",
-      "tslint --fix",
+      "prettier-eslint --write",
       "stylelint --fix",
```

Comment:
It's unrelated to this PR, but let's remove --fix flag from stylelint - it won't work with styled-components

Reply:
@dannytce this `lint-staged` is meant to actually fix the code. If we remove `--fix`, we might as well just remove `stylelint` script altogether, right?

Ps.: I didn't change anything related to `stylelint`, but now that I'm looking at it, I'll add this linter to the `qa` script - which combines all code validation tools.

Reply:
@lucasconstantino I think that `lint-staged` is not just about fixing the code, but mostly not letting you commit 💩code :) Any autofixes there are just additional bonuses. 

The problem with `stylelint --fix` is that it's not working well with styled-componenets at the moment. It basically breaks the whole styles. But I think it's a valid to throw an error (and not commit) if styles are broken anyhow.

Last thing, you can use [our stylelint config](https://github.com/strvcom/code-quality-tools/tree/master/packages/stylelint-config-styled-components) to get rid of a few dependencies :)